USERS.md

```diff
@@ -0,0 +1,28 @@
+# Who is using Karpenter?
+Karpenter has a variety of users and use cases for scaling Kubernetes.
+Many customers want to learn from others who have already implemented Karpenter in their environments.
+
+The following is a self-reported list of users to help identify adoption and points of contact.
+
+## Community
+If you would like to ask question from the community please join the [Karpenter slack channel in the Kubernetes Slack](https://kubernetes.slack.com/archives/C02SFFZSA2K) or join the [Karpenter working group](WORKING_GROUP.md) and fort-nightly calls.
```

Comment:
Definitely great use of fortnight here, but I think we should stick to bi-weekly so someone doesn't have to look it up. 
```suggestion
If you would like to ask question from the community please join the [Karpenter slack channel in the Kubernetes Slack](https://kubernetes.slack.com/archives/C02SFFZSA2K) or join the [Karpenter working group](WORKING_GROUP.md) bi-weekly calls.
```